app/views/admin/users/index.html.erb

```diff
@@ -28,12 +29,14 @@
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td class="email">
+        <td class="email" nowrap>
```

Comment:
The `nowrap` attribute on a td element is obsolete. This should be achieved using CSS.